web/static/name_maps.js

```diff
@@ -415,6 +415,7 @@ export const geojsonTypes = {
   'Bay Area EV Roadmap': 'area',
   'Salt Lake City Region EV Plan': 'area',
   'Truck Stop Locations': ['point', 'other'],
+  'DOE Corridors': ['point', 'other'],
```

Comment:
I think it might make sense to comment this out for now since the DOE corridor effort got put on pause.